infrastructure/Makefile

```diff
@@ -139,9 +139,13 @@ setup-postgresql:
 # List databases
 	@sudo -u postgres psql -c "\l"
 
+reset-db:
+	@sudo -u postgres psql -c "DROP DATABASE IF EXISTS $(POSTGRES_DBNAME);"
+	@sudo -u postgres createdb $(POSTGRES_DBNAME)
+	@sudo -u postgres psql -c "grant all privileges on database $(POSTGRES_DBNAME) to $(POSTGRES_UNAME)"
 
 # Sets up indexer in local mode (to be used with private-net)
-start-indexer:
+start-indexer: reset-db
```

Comment:
why it will reset the DB? I think it's better to continue and have similar commands as we have with the private net for recreating: `setup-*` and `recreate-*`

Reply:
This is recreation only. It will drop the database and create a new one (before starting indexer each time). While i was doing [this](https://www.pivotaltracker.com/story/show/178687692) task using `algorand-indexer`, with each start the indexer wasn't able to catch up properly so i was doing this manually. 

Shall i rename this to `recreate-db`, or close this PR?

Reply:
yes, let's rename to `recreate-db`

Reply:
ok